tests/visualization_tests/test_edf.py

```diff
@@ -11,6 +14,14 @@ def test_target_is_none_and_study_is_multi_obj() -> None:
         plot_edf(study)
 
 
+def _validate(y_values: Sequence[float]) -> None:
+    np_values = np.array(y_values)
+    # This line confirms that the values are monotonically non-decreasing.
```

Comment:
how about this simplification?
```suggestion
    # Confirms that the values are monotonically non-decreasing.
```